Error: Cannot find module 'whatwg-url'

https://github.com/uwdata/arquero/commit/8b4fe94b84c83f5da4651678226a81663df87313 introduced a `whatwg-url` as a dependency and `yarn upgrade arquero@~4.8.6` did not add it.

Please add dependency or remove it if not needed.

EDIT: Hmm, actually, it wasn't added to package, just in the lock file. But it's giving me the following error. Running `yarn cache clean` and reinstall to `@4.8.4` did not fix the problem...

```
node:internal/modules/cjs/loader:941
  const err = new Error(message);
              ^

Error: Cannot find module 'whatwg-url'
Require stack:
- /Users/john/dev/bdb/node_modules/arquero/dist/arquero.node.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:941:15)
    at Function.Module._load (node:internal/modules/cjs/loader:774:27)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/john/dev/bdb/node_modules/arquero/dist/arquero.node.js:9:17)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:201:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:175:25)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
    at async Object.loadESM (node:internal/process/esm_loader:68:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/john/dev/bdb/node_modules/arquero/dist/arquero.node.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "arquero",
-  "version": "4.8.6",
+  "version": "4.8.7",
   "description": "Query processing and transformation of array-backed data tables.",
   "keywords": [
     "data",
@@ -43,11 +43,11 @@
   },
   "devDependencies": {
     "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^13.0.4",
+    "@rollup/plugin-node-resolve": "^13.0.5",
     "eslint": "^7.32.0",
     "esm": "^3.2.25",
     "rimraf": "^3.0.2",
-    "rollup": "^2.56.3",
+    "rollup": "^2.57.0",
     "rollup-plugin-bundle-size": "^1.0.3",
     "rollup-plugin-terser": "^7.0.2",
     "tape": "^5.3.1",

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -10,7 +10,7 @@ function onwarn(warning, defaultHandler) {
 }
 
 const name = 'aq';
-const external = [ 'apache-arrow' ];
+const external = [ 'apache-arrow', 'node-fetch' ];
 const globals = { 'apache-arrow': 'Arrow' };
 const plugins = [
   json(),
@@ -21,14 +21,13 @@ const plugins = [
 export default [
   {
     input: 'src/index-node.js',
-    external,
+    external: ['acorn'].concat(external),
     plugins,
     onwarn,
     output: [
       {
         file: 'dist/arquero.node.js',
         format: 'cjs',
-        globals,
         name
       }
     ]

